src/main/java/uk/gov/justice/digital/domain/DomainExecutor.java

```diff
@@ -57,7 +59,7 @@ public DomainExecutor(JobArguments jobParameters,
     protected void insertTable(TableIdentifier info, Dataset<Row> dataFrame)
             throws DomainExecutorException {
         logger.info("DomainExecutor:: insertTable");
-        String tablePath = storage.getTablePath(info.getBasePath(), info.getSchema(), info.getTable());
+        String tablePath = createValidatedPath(info.getBasePath(), info.getSchema(), info.getTable());
```

Comment:
are we catching the exception IllegalArgumentException here?

Reply:
The intent is to fail fast here. I think this will be fine but we should see how it plays on test but I don't expect there will be any issues here.